posthog/hogql/parser_utils.py

```diff
@@ -0,0 +1,27 @@
+def parse_string_literal(ctx):
```

Comment:
nit: I'd avoid `utils` it becomes a "big bag o' functions"

Is this `parse_string_literal.py` `string_parsing.py` ??

Reply:
`parse_string.py` is now a thing.